flat-manager/index.js

```diff
@@ -12,13 +12,28 @@ const run = (repository, flatManagerUrl, token) => {
   ])
     .then(async () => {
       let buildId = ''
-      const exitCode = await exec.exec('flat-manager-client', [
+      let args = [
         '--token',
-        token,
+        token        
+      ]
+
+      if (end_of_life) {
+        args.push_back(`--end-of-life=${end_of_life}`)
+
+        if (end_of_life_rebase) {
+          args.push_back(`--end-of-life-rebase=${end_of_life_rebase}`)
+        } else {
+          throw Error('end-of-life has to be set if you want to use end-of-life-rebase')
+        }
```

Comment:
This condition is wrong no? you want to throw the error if !end_of_life && end_of_life_rebase.

Reply:
Was perhaps a little too early in the morning :sweat_smile: